velero-plugins/replicaset/restore.go

```diff
@@ -38,27 +38,6 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 	common.SwapContainerImageRefs(replicaSet.Spec.Template.Spec.Containers, backupRegistry, registry, p.Log, input.Restore.Spec.NamespaceMapping)
 	common.SwapContainerImageRefs(replicaSet.Spec.Template.Spec.InitContainers, backupRegistry, registry, p.Log, input.Restore.Spec.NamespaceMapping)
 
-	ownerRefs, err := common.GetOwnerReferences(input.ItemFromBackup)
-	if err != nil {
-		return nil, err
-	}
-
-	annotations := replicaSet.Annotations
-
-	// Don't restore ReplicaSet if owned by Deployment and the deployment was not paused
```

Comment:
Is the  replicaset restored before the deployment?

Reply:
@amastbau Yes -- I just checked the restore priorities list. They're restored right after pods. Deployments come later in the "everything not called out as high priority" group